core/playwright/login.flows.ts

```diff
@@ -0,0 +1,12 @@
+import type { Page } from "@playwright/test";
+
+import { expect } from "@playwright/test";
+
+export const login = async (page: Page, email = "new@pubpub.org", password = "pubpub-new") => {
+	await page.goto("/login");
+	await page.getByLabel("email").fill(email);
+	await page.getByRole("textbox", { name: "password" }).fill(password);
+	await page.getByRole("button", { name: "Sign in" }).click();
+	await page.waitForURL(/.*\/c\/\w+\/stages.*/);
+	await expect(page.getByRole("link", { name: "Workflows" })).toBeVisible();
+};
```

Comment:
and regardless of how you proceed with the suggestions above, could you consolidate the `login` function in `core/playwright/helpers.ts` with this code? turns out the code you deleted in login.spec.ts was already duplicated (just with a different email by default).

Reply:
Because the `expect` calls are not pure assertions (e.g. in many cases they wait for the specified element to become visible) I don't think we can separate them from the `page` interaction and navigation calls. For that reason I think it's best to keep the assertions as part of the shared test instead of explicitly calling them in one or both test suites, because most times we'd just be duplicating the `expect` calls.

So I think we really do need a new kind of module that is neither a test nor a test fixture. I'm referring to them here as "flows". I think in many (or most) cases a flow will be a complete test that was just extracted to share it with the load test suite. But in some cases, like the above login example, a "flow" could be a partial playwright routine that can be re-used between tests.